linode/manager
# M3-5426: Add Cypress integration tests for Linode Managed

## Description 📝

**What does this PR do?**
* Adds integration tests for Linode Managed, including CRUD operations for each tab, as well as some tests for navigation, etc.
* Adds some more MSW handlers for Managed
* Changes the `credentials` property of `ManagedServiceMonitor` to an array of numbers rather than an array of credential objects to match what the API expects
* Upgrades Cypress to `v10.10.0`

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
* To test the MSW handlers, run `yarn up`, enable MSW, and test CRUD operations on each resource type (monitors, contacts, credentials, etc.). I wasn't able to mock all requests but coverage is a bit better than it was.

**How do I run relevant unit or e2e tests?**
```bash
yarn cy:run -s "cypress/e2e/managed/managed-conta
cts.spec.ts,cypress/e2e/managed/managed-credentials.spec.ts,cypress/e2e/managed/managed-monitors
.spec.ts,cypress/e2e/managed/managed-navigation.spec.ts,cypress/e2e/managed/managed-ssh.spec.ts,
cypress/e2e/managed/managed-summary.spec.ts"
```

Alternatively, if you want to run them interactively, run `yarn cy:debug` and manually run each spec in the `managed` directory.
